README.md

```diff
@@ -1 +1,77 @@
-# nvim-package-specification
\ No newline at end of file
+# Plugin metadata specification
+
+The neovim package specification supports a single, top-level package metadata file. This file can be *either* 'plugin.lua' or 'plugin.json'. The file is a superset of the Rockspec File Format and can contain the following fields
+
+* `package` : String, the name of the package
+
+* `version` : String, the version of the package. Should obey semantic versioning conventions, for example `0.1.0`. For all version identifiers, implementation should check for a `version` prefixed with `v` in the git repository, as this is a common convention.
+
+* `specification_version` : String, the current specification version. (0.1.0) at this time.
+
+
+* `source` : Table, the source is a table that contains a `url` field, which points to either the git commit of the current version of the package, or the source tarball.
+
+* `description` : Table, the description is a table that includes the following nested fields:
+	* `summary` : String, a short description of the package, typically less than 100 character long.
+	* `detailed` : String, a long-form description of the package, this should convey the package's principal functionality to the user without being as detailed as the package readme.
+	* `homepage` : This is the homepage of the package, which in most cases will be the GitHub URL.
+	* `license` : This is [SPDX](https://spdx.org/licenses/) license identifier. Dual licensing is indicated via joining the relevant licenses via `/`.
+
+* `dependencies`
+** TBD
+
+* `dependency_sources`
+** TBD
```

Comment:
Is there a reason not to merge these two fields? I would suggest a single `dependencies` list containing elements like:

`name`: String, the canonicalized name of the dependency
`reason`: Optional string, a short description of the purpose served by the dependency
`version`: String, a SemVer version constraint specifying the compatible versions of the dependency
`type`: String, either "plugin" or "rock", specifying if the dependency is a plugin or a Luarocks rock
`source`: Either string, assumed to be the URL of a git repo, or table, as for the top-level `source` key.

Reply:
The only reason is because I was trying to stay consistent with rockspec, so I made this field such that our fields were a superset, and the fields that were shared were compatible with luarocks.